### DISCUSSION TITLE
Help with mesh leveling CR touch gives progressively more negative results the more points it probes

### DISCUSSION THREAD

**Discussion by imsiar** [mriscoc/Ender3V2S1#976] [Q&A]
I have an ender 3v2 neo with 4.22 board prior versions worked great but after trying to instal the latest version i get this strange behaviour: First 10-14 points seem to work fine afterwards every point starts to become more and more negative,  it gives the aspect of giant bulge in the center of my bed. 
  I tried using bilinear and get the same results only this time the first points in front of the bed seem fine and gets progressively more negative the more points it probes.
  I've tried flashing countless version with every possible modification that i could think of messing with high speed bluetooth, mesh settings, UBL, Probe settings and i Always get the same results.
  When i flash back to my previous version the problem is resolved and my mesh is perfect.
  Does anyone have any idea what could be causing this and what else should I try?

**Comment by mriscoc** [comment]
That could be the effect of a probe compensation added in latest version, that seems to improve the mesh in many devices but maybe not in yours. I will try to make it configurable on next versions.

**Reply by imsiar** [reply]
Awesome thank you I'll stick to the older version until then

**Comment by mriscoc** [comment]
Possible solution: https://github.com/mriscoc/Ender3V2S1/discussions/1021

**Reply by imsiar** [reply]
Thanks a lot as soon as i unpack my 3d printer I'll give it a try

**Reply by bagsofcole** [reply]
that worked for my Neo

**Comment by uiJLKJ** [comment]
@imsiar where can I find an older version ?
I have a similar problem with bed tramming, I have to auto home between every manual tramming.
![IMG_20230815_000624](https://github.com/mriscoc/Ender3V2S1/assets/91351706/3de07dca-531e-41fa-8027-36b52e9a82e8)

I'm using the same function right after the first time
![IMG_20230815_000557](https://github.com/mriscoc/Ender3V2S1/assets/91351706/c7579e2b-8a1e-4692-ba06-dd62865da7dc)

After auto home
![IMG_20230815_000550](https://github.com/mriscoc/Ender3V2S1/assets/91351706/12e7eb18-2aab-47b0-9b88-3e29336ad563)

Edit : sorry I have read the thread before going to the bed yesterday, I didn't see @mriscoc 's answer
I'm still looking after an older version for now as I couldn't resolve my problem

**Reply by imsiar** [reply]
https://github.com/mriscoc/Ender3V2S1/releases
Scroll down till you reach one you like. I reverted to the march one "boreal spring"

**Comment by uiJLKJ** [comment]
My bad, I feel really dumb about that, after a few tests I realized my z rob started to unscrew due to vibrations...
